crypto/src/lib.rs

```diff
@@ -208,6 +212,31 @@ pub trait Crypto {
         info: &[u8],
     ) -> Result<Self::Cdi, CryptoError>;
 
+    /// Retrieve an exported CDI Handle from a CDI.
+    /// If the CDI has already been exported, the same handle is returned.
+    /// If the number of CDIs that can be exported has reached its limit,
+    /// a `CryptoError` is returned.
+    ///
+    /// # Arguments
+    ///
+    /// * `cdi` - The CDI to request a handle for.
+    fn get_exported_cdi_handle(
+        &mut self,
+        cdi: &Self::Cdi,
+    ) -> Result<[u8; MAX_EXPORTED_CDI_SIZE], CryptoError>;
+
+    /// Retrieve a CDI from an exported CDI Handle.
+    ///
+    /// If the handle is not associated with a CDI, `None` is returned.
+    ///
+    /// # Arguments
+    ///
+    /// * `exported_cdi_handle` - The handle to the CDI.
+    fn get_cdi_from_exported_handle(
```

Comment:
Isn't this internal to Caliptra RT FW since SIGN_WITH_EXPORTED is a Caliptra command? I'm not sure DPE needs to be able to do this lookup.

Reply:
That's a fair point, I'll remove this from the trait.